src/platform/macos/window.rs

```diff
@@ -452,9 +452,15 @@ impl WindowDelegate {
         unsafe {
             let delegate = IdRef::new(msg_send![WindowDelegate::class(), new]);
 
+            // setDelegate uses autorelease on objects,
+            // so need autorelease
+            let autoreleasepool = NSAutoreleasePool::new(cocoa::base::nil);
```

Comment:
Is there any reason you went with `cocoa::base::nil` instead of using the imported `nil`? It's a minor gripe, but having that consistent makes it easier for people to tell the same type is being used throughout.